metabase/metabase
# Add dynamic scatter plot

### Description

- Adds dynamic scatter plot built with ECharts
- Fixes bubble size:
  - Default bubble size did not match previous implementation
  - Specifying bubble size on charts with a breakout did not render anything

### How to verify

Verify dynamic scatter plots built with ECharts look as previous dc.js implementation.
Check all of the following dimensions:
- Single metric 
- Multiple metric
- Single metric + breakout
- Single metric with bubble size column
- Multiple metric with bubble size column
- Single metric + breakout with bubble size column

### Demo

**ECharts**
<img width="1717" alt="Screenshot 2023-12-28 at 7 04 34 PM" src="https://github.com/metabase/metabase/assets/14301985/7ce58d07-349a-4656-9a70-525b13254226">

**dc.js**
<img width="1719" alt="Screenshot 2023-12-28 at 7 04 47 PM" src="https://github.com/metabase/metabase/assets/14301985/a55e0806-b628-4edb-a340-0f5e3da0f9a4">

### Checklist
